.github/README.md

```diff
@@ -0,0 +1,183 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books <a name="about-project"></a>
+
+ This is a Awesome books: plain JavaScript with objects project in the Microverse program to test my understanding in different ways to create objects in JavaScript, and Create and access properties and methods of JavaScript objects.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+    <li>BOOTSTRAPL</li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Responsive design**
+- **Used only Bootstrap, CSS, html and Javascript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://hanasabih.github.io/awesome-books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A personal computer connected to the internet
+- A Modern/Updated web browser
+- Have a local version control like git installed on your computer and a Github account
+- Code editor installed on your computer
+
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+    cd your-desired-folder
+    git clone https://github.com/HanaSabih/awesome-books.git
+```
+
+
+### Install
+
+Install this project with:
+```sh
+    cd your-desired-folder
+    git clone https://github.com/HanaSabih/awesome-books.git
+```
+
+
+
+
+### Usage
+
+To run the project, open index.html with your preferred browser.
+
+
+
+### Run tests
+
+To run tests, run the following command in your `git bash` or CLI:
+- `npx hint .` : for the html codebase and use of best practices.
+- `npx stylelint "**/*.{css,scss}" `: for the css codebase and use of best practices. you can add.
+- `npx eslint . `: for the JavaScript codebase and use of best practices.
+
+
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Hana Sabih**
+
+- GitHub: [@githubhandle](https://github.com/HanaSabih)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hana-sabih/)
+
```

Comment:
🌸  So far your readme file is well-edited and descriptive. Be sure to update this authors' section to include yourself